dev/com.ibm.ws.classloading/resources/OSGI-INF/l10n/classloader.properties

```diff
@@ -25,10 +25,15 @@ classloader.delegation.desc=Controls whether parent classloader is used before o
 # Option descriptions are not supported in Metatype spec but we may extend for that
 parentFirst=Parent first
 parentLast=Parent last
-# libraryRefs
+# Private libraryRefs
 classloader.library.ref=Library
 classloader.library.ref$Ref=Library references
-classloader.library.ref.desc=List of library references. Library class instances are unique to this classloader, independent of class instances from other classloaders.
+classloader.library.ref.desc=List of library references. Library class instances are unique to this classloader, independent of class instances from other classloaders. The referenced library class path is searched after searching this classloaders own class path.
+
+# Patch libraryRefs
+classloader.patch.library.ref=Patch Library
+classloader.patch.library.ref$Ref=Patch Library references
+classloader.patch.library.ref.desc=List of library references. Library class instances are unique to this classloader, independent of class instances from other classloaders. The referenced library class path is searched before searching this classloaders own class path.
```

Comment:
Change to: classloader.patch.library.ref.desc=List of library references. Library class instances are unique to the classloader element and independent of class instances from other classloader elements. The referenced library class path is searched before the classloader class path is searched.